### DISCUSSION TITLE
能不能在MOD管理中增加筛选功能与标记功能

### DISCUSSION THREAD

**Discussion by RiverSDream** [Meloong-Git/PCL#818] [提问] [answered]
制作起来可能比较麻烦，但是如果完成了会对未开启版本隔离的用户有很大程度的帮助，即：
我可以一键筛选出所有符合当前版本(例:1.18.2-Fabric 0.14.9)要求的MOD
我想到了一个方案，即：这个功能需要用户手动开启，初次运行时扫描MOD列表，扫描不出版本的需要用户手动编辑

标记功能：这个功能更为重要，因为我总是记不得哪些MOD是另一个MOD的前置模组，开启标记功能之后，可以提醒我；或者可以设置为标记之后，启用MOD本体后，自动启用其所有的前置模组。

**Comment by WForst-Breeze** [comment, marked as answer]
操作有点过于烦琐了……没太大必要，因为一般玩 mod 整合包的玩家都会开版本隔离，不开版本隔离的人很少。况且如何筛选 mod 的版本和其前置也是个问题，必然会有不准确的筛选结果，到时候又会出一大堆问题
不过你也可以去交个[ **新功能提案** ](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)看看开发者怎么想，只是**我觉得**可能被拒绝的几率比较大就是了，感谢反馈

**Reply by RiverSDream** [reply]
> 操作有点过于烦琐了……没太大必要，因为一般玩 mod 整合包的玩家都会开版本隔离，不开版本隔离的人很少。况且如何筛选 mod 的版本和其前置也是个问题，必然会有不准确的筛选结果，到时候又会出一大堆问题 不过你也可以去交个[ **新功能提案** ](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)看看开发者怎么想，只是**我觉得**可能被拒绝的几率比较大就是了，感谢反馈

感谢，我在想了想之后保留了后一个功能，并放到了新功能提案
